apps/central-scan/backend/src/store.ts

```diff
@@ -47,6 +47,76 @@ const debug = makeDebug('scan:store');
 
 const SchemaPath = join(__dirname, '../schema.sql');
 
+const getSheetsBaseQuery = `
+  select
+    sheets.id as id,
+    batches.id as batchId,
+    batches.label as batchLabel,
+    front_interpretation_json as frontInterpretationJson,
+    back_interpretation_json as backInterpretationJson,
+    front_image_path as frontImagePath,
+    back_image_path as backImagePath,
+    requires_adjudication as requiresAdjudication,
+    finished_adjudication_at as finishedAdjudicationAt,
+    sheets.deleted_at as deletedAt,
+    row_number() over (partition by batches.id order by sheets.created_at) indexInBatch
+  from sheets left join batches on
+    sheets.batch_id = batches.id
+`;
+
+interface SheetRow {
+  id: string;
+  batchId: string;
+  batchLabel: string | null;
+  frontInterpretationJson: string;
+  backInterpretationJson: string;
+  frontImagePath: string;
+  backImagePath: string;
+  requiresAdjudication: 0 | 1;
+  finishedAdjudicationAt: Iso8601Timestamp | null;
+  deletedAt: Iso8601Timestamp | null;
+  indexInBatch: number;
+}
+
+function sheetRowToAcceptedSheet(row: SheetRow): AcceptedSheet {
+  assert(row.deletedAt === null);
+  return {
+    type: 'accepted',
+    id: row.id,
+    batchId: row.batchId,
+    batchLabel: row.batchLabel ?? undefined,
+    interpretation: mapSheet(
+      [row.frontInterpretationJson, row.backInterpretationJson],
+      (json) => safeParseJson(json, PageInterpretationSchema).unsafeUnwrap()
+    ),
+    frontImagePath: row.frontImagePath,
+    backImagePath: row.backImagePath,
+    indexInBatch: row.indexInBatch,
+  };
+}
+
+function sheetRowToRejectedSheet(row: SheetRow): RejectedSheet {
+  assert(row.deletedAt !== null);
+  return {
+    type: 'rejected',
+    id: row.id,
+    frontImagePath: row.frontImagePath,
+    backImagePath: row.backImagePath,
+  };
+}
+
+function sheetRowToSheet(row: SheetRow): Sheet {
+  assert(
+    row.requiresAdjudication === 0 ||
+      row.finishedAdjudicationAt !== null ||
+      row.deletedAt !== null,
+    'Every sheet requiring review should have been either accepted or rejected'
+  );
```

Comment:
The VxCentralScan UX guarantees this - sheets requiring review have to be accepted or rejected before a batch is considered complete. And if someone shuts the machine down mid-adjudication, on boot, incomplete batches are cleaned up:

https://github.com/votingworks/vxsuite/blob/8b41fdada9e3e8e961b9a4f44f93e6475052a836/apps/central-scan/backend/src/server.ts#L59

Reply:
We've discussed changing how central scan does adjudication - maybe add a comment with a bit of this explanation so that anyone changing this in the future sees the context?

Reply:
Good call! https://github.com/votingworks/vxsuite/pull/4012/commits/0db5543ceca28275706c39d9ecd23410fc221804